css/design_system/components/footer.scss

```diff
@@ -0,0 +1,110 @@
+ /**
+ * @title Simple Footer
+ * @category components
+ * @component_type footer
+ * @value s-footer
+ * @description Simple footer with paragraph text only. The <code>.s-footer-content</code> class should be used on the container that wraps content.
+ *
+ * @example
+ * <footer class="s-footer">
+ *  <div class="s-footer-content">
+ *    <p>Turing School of Software and Design, a Colorado Non-Profit Organization.</p>
+ *  </div>
+ * </footer>
+ */
+
+.s-footer {
+  @extend .s-p-4;
+  @extend .s-bg-gray-800;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.s-footer-content {
+  * {
+    @extend .s-my-0;
+    @extend .s-text-gray-100;
+    @extend .s-text-start;
+  }
+}
+
+ /**
+ * @title Footer with Links
+ * @category components
+ * @component_type footer
+ * @description Footer with paragraph text and one set of links. Use of the <code>.s-footer-right</code> and <code>.s-footer-link</code> classes ensure proper spacing between links.
+ *
+ * @example
+ * <footer class="s-footer">
+ *  <div class="s-footer-content">
+ *    <p>Turing School of Software and Design, a Colorado Non-Profit Organization.</p>
+ *  </div>
+ *  <div class="s-footer-right">
+ *    <a class="s-footer-link" href="/privacy">Privacy Policy</a>
+ *    <a class="s-footer-link" href="https://turing.edu/programs/" target="blank">Programs</a>
+ *    <a class="s-footer-link" href="https://turing.edu/try-coding/">Try Coding</a>
+ *  </div>
+ * </footer>
+ */
+
+.s-footer-left {
+  > :first-child {
+    @extend .s-ml-0;
+    @extend .s-pl-0;
+  }
+}
+
+.s-footer-right {
+  > :last-child {
+    @extend .s-mr-0;
+    @extend .s-pr-0;
+  }
+}
+
+.s-footer-link {
+  @extend .s-text-uppercase;
+  @extend .s-text-body-sm;
+  @extend .s-font-bold;
+  @extend .s-text-gray-100;
+  @extend .s-pr-4;
+  @extend .s-pl-2;
+  text-decoration: none;
+  letter-spacing: var(--s-space-1);
+  cursor: pointer;
+}
+
+.s-footer-link:hover {
+  @extend .s-text-white;
+}
```

Comment:
Not critical, but just for future reference - in SCSS you can nest modifier styles like `:hover` and use the `&` symbol to stand in for the current selector, so this is equivalent:

```suggestion
.s-footer-link {
  @extend .s-text-uppercase;
  @extend .s-text-body-sm;
  @extend .s-font-bold;
  @extend .s-text-gray-100;
  @extend .s-pr-4;
  @extend .s-pl-2;
  text-decoration: none;
  letter-spacing: var(--s-space-1);
  cursor: pointer;
  
  &:hover {
    @extend .s-text-white;
  }
}
```

Reply:
Awesome, thank you!!